src/enc/backward_references/hash_to_binary_tree.rs

```diff
@@ -376,12 +376,16 @@ impl<'a> BackwardMatchMut<'a> {
     pub fn set_length_and_code(&mut self, data: u32) {
         *self.0 = u64::from((*self.0) as u32) | (u64::from(data) << 32);
     }
-}
-
-#[inline(always)]
-pub fn InitBackwardMatch(xself: &mut BackwardMatchMut, dist: usize, len: usize) {
-    xself.set_distance(dist as u32);
-    xself.set_length_and_code((len << 5) as u32);
+    #[inline(always)]
+    pub fn init(&mut self, dist: usize, len: usize) {
+        self.set_distance(dist as u32);
+        self.set_length_and_code((len << 5) as u32);
+    }
+    #[inline(always)]
+    pub(crate) fn init_backward(&mut self, dist: usize, len: usize, len_code: usize) {
```

Comment:
should this be init_dictionary instead of init_backward?

Reply:
good catch, fixed